crud-module/index.js

```diff
@@ -0,0 +1,145 @@
+            name: 'addAngularModule',
+            message: 'Would you like to generate the Angular Module?',
+            default: true
+        }, {
+            type: 'confirm',
+            name: 'addExpressModule',
+            message: 'Would you like to generate the Express Module?',
+            default: true
+        }];
+
+        this.prompt(prompts, function(props) {
+            this.addAngularModule = props.addAngularModule;
+            this.addExpressModule = props.addExpressModule;
+
+            done();
+        }.bind(this));
+    },
+
+    copyTemplates: function() {
+
+        var files = {
+            'module.client.module.js' : folderPath  + "/client/" + this.pluralModuleName + ".client.module.js",
+            'module.client.config.js' : folderPath + "/client/config/" + this.pluralModuleName + ".client.config.js",
+            'module.client.controller.js' : folderPath  + "/client/controllers/" + this.pluralModuleName + ".client.controller.js",
+            'module.client.routes.js' : folderPath  + "/client/config/" + this.pluralModuleName + ".client.routes.js",
+            'module.client.service.js' : folderPath  + "/client/services/" + this.pluralModuleName + ".client.service.js",
+            'views/create.client.view.html' : folderPath  + "/client/views/create-" + this.moduleName + ".client.view.html",
+            'views/edit.client.view.html' : folderPath  + "/client/views/edit-" + this.moduleName + ".client.view.html",
+            'views/list.client.view.html' : folderPath  + "/client/views/list-" + this.pluralModuleName + ".client.view.html",
+            'views/view.client.view.html' : folderPath  + "/client/views/view-" + this.moduleName + ".client.view.html",
+            'module.server.config.js' : folderPath  + "/server/config/" + this.pluralModuleName + ".server.config.js",
+            'module.server.controller.js' : folderPath  + "/server/controllers/" + this.pluralModuleName + ".server.controller.js",
+            'module.server.model.js' : folderPath  + "/server/models/" + this.moduleName + ".server.model.js",
+            'module.server.policy.js' : folderPath  + "/server/policies/" + this.pluralModuleName + ".server.policy.js",
+            'module.server.routes.js' : folderPath  + "/server/routes/" + this.pluralModuleName + ".server.routes.js"
+        };
+
+        for(var template in files) {
+            this.fs.copyTpl(
+                this.templatePath(template),
+                this.destinationPath(files[template]),
+                {
+                    moduleName: this.name,
+                    capitalizedModuleName: this.capitalizedModuleName,
+                    pluralModuleName: this.pluralModuleName,
+                    pluralCapitalizedModuleName: this.pluralCapitalizedModuleName
+                }
+            );
+        }
+    },
+
+    removeAngularModule: function () {
+        if(!this.addAngularModule) {
+            exec('rm -rf ' + folderPath + 'client')
```

Comment:
`rm -rf` won't work on Windows, as I recall. I'm actually on windows but I have *cygwin* installed, so it works for me. I can't remember what the resolution to this was previously with the main generator.

@codydaig How was this reconciled in the current generator?